forgerock-authenticator/src/main/java/org/forgerock/android/auth/FRAPolicyEvaluator.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2023 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import org.forgerock.android.auth.policy.FRAPolicy;
+import org.forgerock.android.auth.policy.BiometricAvailablePolicy;
+import org.forgerock.android.auth.policy.DeviceTamperingPolicy;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The Policy Evaluator is used by the SDK to enforce Policy rules, such as Device Tampering Policy.
+ * It consist of one or more {@link FRAPolicy} objects. Each Policy contain instructions that
+ * determine whether it comply to a particular condition at a particular time.
+ */
+public class FRAPolicyEvaluator {
+
+    private final List<FRAPolicy> policies;
+
+    private static final String TAG = FRAPolicyEvaluator.class.getSimpleName();
+
+    public static final List<FRAPolicy> DEFAULT_POLICIES = new ArrayList<>();
+
+    static {
+        DEFAULT_POLICIES.add(new DeviceTamperingPolicy());
+        DEFAULT_POLICIES.add(new BiometricAvailablePolicy());
+    }
+
+    private FRAPolicy nonCompliancePolicy;
```

Comment:
Maintaining state here may be dangerous, this makes the Evaluator can use only evaluate once.

Reply:
Good point, but I think we could clean this attribute when starting a new evaluation instead of remove this property. How that sounds?

Reply:
With that, may not thread safe.

Reply:
Done, created a Result object to replace boolean as result